src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.html

```diff
@@ -53,12 +49,27 @@
           href="#/dashboard/create"
           aria-label="Create new dashboard"
           data-test-subj="newDashboardLink"
-          ng-hide="listingController.getSelectedItemsCount() > 0"
+          ng-if="listingController.getSelectedItemsCount() === 0"
           tooltip="Create new dashboard"
         >
           <span aria-hidden="true" class="kuiButton__icon kuiIcon fa-plus"></span>
         </a>
       </div>
+
+      <div class="kuiToolBarSection">
```

Comment:
Awesome!  Only one issue is that if you select all, looks like it selects everything, not just the first page, so if you do select all and delete, it'll delete everything.  I don't think this is what we want.